code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1127,3 +1128,107 @@
 			return ..()
 
 #undef BROOM_PUSH_LIMIT
+
+/obj/item/twohanded/supermatter  //Supermatter Halberd, used by Oblivion Enforcers
+	name = "supermatter halberd"
+	desc = "The revered weapon of Oblivion Enforcers, used to enforce the Order's will."
+	icon_state = "smhalberd0"
+	force = 5
+	sharp = TRUE
+	damtype = BURN
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = SLOT_BACK
+	throwforce = 15
+	force_unwielded = 5
+	force_wielded = 40
+	toolspeed = 0.25
+	attack_verb = list("enlightened", "enforced", "cleaved", "stabbed", "whacked")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	resistance_flags = FIRE_PROOF
+	var/static/list/obliteration_targets = list(/turf/simulated/wall, /obj/machinery/door/airlock)
+	/// Whether we'll knockdown on hit
+	var/charged = TRUE
+
+/obj/item/twohanded/supermatter/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_FORCES_OPEN_DOORS_ITEM, ROUNDSTART_TRAIT)
```

Comment:
Does it really need the trait to open doors if it can dust them?

Reply:
if you want to be nice as an adminbus enforcer yeah